misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -68,37 +74,40 @@
      * @param groups          List of groups to be included
      */
     public void runTest(String sourceRoot, Path[] sourceFilePaths, List<String> groups) {
+
```

Comment:
Why are we adding a new line just after the function signature before starting the function body?

Reply:
This comes from wso2 Idea formatting rules.